scriptmodules/emulators/openmsx.sh

```diff
@@ -54,4 +54,8 @@ function configure_openmsx() {
 
     addEmulator 0 "$md_id" "msx" "$md_inst/bin/openmsx %ROM%"
     addSystem "msx"
+
+    mkUserDir "$home/.openMSX/share/systemroms"
```

Comment:
This could fail if the folder doesn't exist?

Probably best to symlink the entire config folder as we do elsewhere also.

Reply:
No, `mkUserDir` uses `mkdir -p`, which created the parent folders if it's not necessary.

https://github.com/RetroPie/RetroPie-Setup/blob/87839ed171dc3fa99adef80869fa152e19452091/scriptmodules/helpers.sh#L468-L475

Reply:
True - I should know this. However it will make them as root and your chosen isn't $user:$user

But I still prefer the main config symlinked to match elsewhere - and moveCondigDir is basically designed to do that, rather than a subfolder.

This is an old module as I guess it got overlooked.